include/seqan3/search/fm_index/fm_index.hpp

```diff
@@ -223,6 +234,9 @@ class fm_index
      * No guarantees.
      */
     void construct(text_t const & text)
+        //!\cond
+        requires !is_collection
```

Comment:
trailing location?

Reply:
Since there is no template I think the trailing location looks better here